src/main/java/it/pagopa/transactions/services/TransactionsService.java

```diff
@@ -137,10 +141,18 @@ public Mono<TransactionInfoDto> getTransactionInfo(String transactionId) {
                                                         .amount(noticeCode.getAmount())
                                                         .reason(noticeCode.getDescription())
                                                         .paymentToken(noticeCode.getPaymentToken())
-                                                        .authToken(null)
                                                         .rptId(noticeCode.getRptId())
                                         ).toList()
                                 )
+                                .amountTotal(transaction.getAmountTotal())
+                                .feeTotal(transaction.getFeeTotal())
+                                // FIXME in the next release of commons library we will delete the null checking
+                                .origin(
+                                        TransactionInfoDto.OriginEnum.valueOf(
+                                                transaction.getOrigin() == null ? "UNKNOWN"
```

Comment:
for an invalid origin better to block the transaction, what do you think about it?

Reply:
We won't have invalid origin (e.g. null). In the next release of commons library it will be filled with UNKNOW when no values are available. I need to put here this check to make test success. With the next commons release this check will be deleted